compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/cfa/FirReturnsImpliesAnalyzer.kt

```diff
@@ -202,6 +202,9 @@ object FirReturnsImpliesAnalyzer : FirControlFlowChecker() {
     fun KotlinTypeMarker.isSupertypeOf(context: TypeCheckerProviderContext, type: KotlinTypeMarker?) =
         type != null && AbstractTypeChecker.isSubtypeOf(context, type, this)
 
+    fun KotlinTypeMarker.isSubtypeOf(context: TypeCheckerProviderContext, type: KotlinTypeMarker?) =
```

Comment:
Probably it would be a good idea to move this and previous function in some other place

Reply:
moved to FirHelpers.kt